apps/lfx-one/src/app/modules/dashboards/org/org-meetings/components/org-meetings-time-range/org-meetings-time-range.component.html

```
@@ -0,0 +1,47 @@
+<!-- Copyright The Linux Foundation and each contributor to LFX. -->
+<!-- SPDX-License-Identifier: MIT -->
+
+<button
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)